qa/multi-cluster-search/build.gradle

```diff
@@ -64,6 +64,7 @@ BuildParams.bwcVersions.withWireCompatible(ccsSupportedVersion) { bwcVersion, ba
     doFirst {
       nonInputProperties.systemProperty('tests.rest.suite', 'multi_cluster')
       nonInputProperties.systemProperty('tests.rest.cluster', localCluster.map(c -> c.allHttpSocketURI.join(",")))
+      nonInputProperties.systemProperty('tests.rest.remote-cluster-version', bwcVersion.toString())
```

Comment:
Of course, that was my initial plan, but talking to Nhat about this we both thought that since we control the version of the remote cluster here completely this would just complicate determining the remote version unnecessarily. I think we would need to pass in the remote clusters transport URIs to the MultiClusterSearchYamlTestSuiteIT, create a client for the "_cat_nodes" call and then use that. But since we know the version here already I'm not convinced about what benefit this would give us.

Reply:
I see what you mean. right the two can't get out of sync because this one determines which remote cluster we test against. I am fine with this. Nit: should we use remote_cluster_version (aligned with the multi_cluster above)?